student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(_clasroom, age, name, parent_permission)
```

Comment:
- [ ] in this class the `name` and `parent_permission` should be assigned a default value as well.